app/controllers/tag_controller.rb

```diff
@@ -514,10 +508,14 @@ def stats
   end
 
   def comments
+    get_wiki_title
+    fetch_counts
+    @title = params[:id]
```

Comment:
You also need to add the `@contributor_count`, `@wildcard` and `@qids`. The `@qids` should come before you call `fetch_counts` since that's where we need it. All of the instance variables I mentioned can be found in the suggestion I made for `Tag#comments`.